docs/UserGuide.md

```diff
@@ -384,6 +385,9 @@ Alternatively, you can use csv config files to customize the analysis as before
 * `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
 * `author-config.csv`: Add one row for each author in each repo you want to analyze
 
+## Using RepoSense with Netlify
+
+You can use Netlify to run RepoSense online with your configured files. This will yield easy access to the application, saving you the time and effort of setup and downloading the jar file every time you want to use the application. Further instructions can be found on this [guide](UsingNetlifyGuide.md).
```

Comment:
Maybe try this to make it sound more appealing to the readers, the configured files etc can come later in the guide.

Using Netlify, and a fork of this repo, allows you to quickly get started online and enjoy a real time RepoSense report on your repository ...

Reply:
Have updated accordingly